paper/paper.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 'PyMilo: A Python Library for ML I/O'
+tags:
+  - Python
+  - Machine Learning 
+  - Model Deployment
+  - Model Serialization
+  - Transparency
+  - MLOPS
+authors:
+  - name: AmirHosein Rostami
+    orcid: 0009-0000-0638-2263
+    corresponding: true
+    affiliation: 1 
+  - name: Sepand Haghighi
+    orcid: 0000-0001-9450-2375
+    corresponding: false
+    affiliation: 1
+  - name: Sadra Sabouri
+    orcid: 0000-0003-1047-2346
+    corresponding: false
+    affiliation: 1
+  - name: Alireza Zolanvari
+    orcid: 0000-0003-2367-8343
+    corresponding: false
+    affiliation: 1
+affiliations:
+ - name: Open Science Lab
+   index: 1
+
+date: 10 June 2025
+bibliography: paper.bib
+---
+
+# Summary
```

Comment:
I think the figure 1 can be imported here, rather than that I would not suggest any other figure.